### PR TITLE
Document Rust crate most closely corresponding to ICU encoding conversion

### PR DESCRIPTION
**Pull request by hsivonen**
* encoding_rs tracks the Encoding Standard as its scope and conformance target
* Marking as out of scope per earlier meeting

### CODE REVIEW

**Comment by sffc** [comment]
The CLA is still currently configured for the Unicode License.  You can either do what you need to do to sign the CLA, or wait until we roll in the Apache license.

**Comment by srl295** [comment]
> The CLA is still currently configured for the Unicode License. You can either do what you need to do to sign the CLA, or wait until we roll in the Apache license.

Might need an issue for this… the assistant is setup to be org-wide. Investigate on https://cla-assistant.io/ ?

File: docs/ecosystem.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Where multiple maintainers are listed, the first name is the primary maintainer:
 | icu::BreakIterator    | [unicode-segmentation](https://docs.rs/unicode-segmentation/)  | Manish / unicode-rs | No               | Does not support line segmentation      |
 | icu::Collator         | Early Google POC (?)                                           | Shane?              | ?                |                                         |
 | icu::NumberFormatter  | Early Google POC (?)                                           | Shane?              | ?                |                                         |
-| icu::CharConversion   | -                                                              |                     |                  |                                         |
+| icu::CharConversion   | [encoding_rs](https://docs.rs/crate/encoding_rs/)              | Henri               | Encoding Standard| Out of scope for OmnICU                 |
```
Comment (sffc) [inline docs/ecosystem.md:20]: ICU for X

Comment (hsivonen) [reply docs/ecosystem.md]: Fixed. Thanks.

**Pull request merged**